### PR TITLE
add waitForUrlToContain command

### PR DESCRIPTION
**Pull request by ellenmobify**
Status: **Ready for Review** 
Owner: Ellen 
Reviewers: @RobotRogue @twangtina @gsaray @lilyzh @mobify-derrick @marlowpayne 

## Changes
- Adds command `.waitForUrlToContain('url', milliseconds, timeout, messages, callback)` that waits for `milliseconds` milliseconds for the page URL to contain `url`. Default is to wait 10000 ms. 

## Todos:
- [ ] Engineer +1 

### Feedback:
- [x] Increase default timeout to 10000

## How to Test
- in your `package.json`, `"nightwatch-commands": "git://github.com/mobify/nightwatch-commands.git#waitForUrlToContain",`

### CODE REVIEW

**Comment by mobify-derrick** [comment]
Lets add a test for this please

File: commands/waitForUrlToContain.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            timeout: 'Timed out while waiting for url after '
+        };
+    }
+
+    timeout = timeout && typeof (timeout) !== 'function' && typeof (timeout) !== 'object' ? timeout : 0;
+
+    this.startTimer = new Date().getTime();
+    this.cb = callback || function() {
+    };
+    this.ms = milliseconds || 1000;
```
Comment (mobify-derrick) [inline commands/waitForUrlToContain.js]: lets bump the default timeout to 10000

**Comment by marlowpayne** [comment]
A couple pretty general questions for `nightwatch-commands`:
- Should we be following gitflow for this repo? (PR into a `develop` branch which we create releases from that merge into master and get NPM published)
- Should we be linting the code in this repo? (If yes, it would probably be good to bring the linting up-to-date with our standard of using `eslint` for our JS)

**Comment by mobify-derrick** [comment]
@marlowpayne great questions.

1. Yes you are right, we should be doing that. Going forward we will make sure that happens.
2. Also a great suggestion, however I think we can create a separate PR specifically for adding linting and bringing the files up to speed.

**Comment by marlowpayne** [comment]
@mobify-derrick awesome. I totally agree that a separate PR will be needed for linting + other enhancements. Should this PR follow gitflow then? We can add linting and/or other changes into the new release in `develop`.

Another thing: This repo has some tests written for it + a `circle.yml` file, but they're broken and CircleCI isn't integrated into PRs. We should probably look into that as an enhancement too.

**Comment by mobify-derrick** [comment]
@marlowpayne you are probably right, @ellenmobify lets get this PR following gitflow.

Ellen and I were discussing the tests, we need to get them working again. We'll probably engage you for next release for some help on them. Tests for assertions work however the commands do not. I would suggest that this is also a separate PR.

For now I am testing this manually

**Comment by marlowpayne** [comment]
@mobify-derrick all sounds good. I'll also start looking into how easy / painful the catch-up with the `eslint` standard would be.

**Comment by marlowpayne** [comment]
Closing this PR so that it can migrate to one for gitflow.

**Pull request closed without merging**